docs/user/addons.rst

```diff
@@ -0,0 +1,53 @@
+Read the Docs Addons
+====================
+
+**Read the Docs Addons** is a group of features for documentation readers and maintainers that you can add to any documentation set hosted on Read the Docs:
+
+:doc:`Traffic analytics </analytics>`
+    explore pageviews for your docs
+
+:doc:`DocDiff </pull-requests>`
+    highlight changed output from pull requests
+
+:doc:`Pull request notification </pull-requests>`
+    notify readers that they are reading docs from a pull request
+
+:doc:`Flyout </flyout-menu>`
+    easily switch between versions and translations
+
+:doc:`Non-stable notification </notifications>`
+    notify readers that they are reading docs from a non stable release
+
+:doc:`Latest version notification </notifications>`
+    notify readers that they are reading docs from an older version
+
+:doc:`Search as you type </server-side-search/index>`
+    get search results faster
+
+Enabling Read the Docs Addons
+-----------------------------
+
+All projects using the ``build.command`` :ref:`build command <config-file/v2:build.commands>` are already using the Addons, other projects can enable them by following these steps:
+
```

Comment:
Since we plan to enable Read the Docs Addons by default in the near future (https://github.com/readthedocs/addons/issues/72#issuecomment-2200390705), I'd probably add a note here communicating this.

```suggestion

.. note::

   We plan to enable Read the Docs Addons by default for all projects in the near future.
   Please, contact us if you found any issue related to addons so we can fix it before that.
```

Feel free to re-write this note. I wrote it here just as an example.